src/tools/devtools/memory.md

```diff
@@ -46,6 +46,17 @@ is no longer used (see [Dart garbage collection][]).
 
 [Dart garbage collection]: {{site.medium}}/flutter/flutter-dont-fear-the-garbage-collector-d69b3ff1ca30
 
+### Object types
+
+#### Disposable object
+
+A Dart object, that has method `dispose()`, which, by de-facto convention, is expected to be invoked,
+when the object is not needed any more, to avoid memory leaks.
+
+#### Memory-risky object
+
+A Dart object, that, if not disposed or disposed but not GCed, may cause a memory leak.
```

Comment:
Can we even remove "might"? Doesn't it for sure cause a memory leak?

```suggestion
A Dart object that is not disposed, or is disposed but not GCed, might cause a memory leak.
```

Reply:
Nope. It causes memory leak if only not disposed or not GCed.
It is ok, if it is disposed and GCed.